homeassistant/components/http/view.py

```diff
@@ -94,11 +94,9 @@ def register(self, app: web.Application, router: web.UrlDispatcher) -> None:
             for url in urls:
                 routes.append(router.add_route(method, url, handler))
 
-        if not self.cors_allowed:
-            return
-
+        add_cors = app["allow_cors"] if self.cors_allowed else app["add_cors"]
```

Comment:
Can we make these keys a bit more descriptive

- `allow_cors` -> `allow_all_cors`
- `add_cors` -> `allow_configured_cors`